config/prod.secret.exs

```diff
@@ -0,0 +1,20 @@
+use Mix.Config
```

Comment:
prod.secret.exs should never be under git. it contains sensative information.
have a look at https://hexdocs.pm/phoenix/heroku.html#making-our-project-ready-for-heroku.